docs/metrics/customizing-the-sdk/README.md

```diff
@@ -245,6 +245,63 @@ within the maximum number of buckets defined by `MaxSize`. The default
         new Base2ExponentialBucketHistogramConfiguration { MaxSize = 40 })
 ```
 
+#### Produce multiple metric streams from single instrument
+
+When an instrument matches multiple views, it can generate multiple metric
```

Comment:
> When an instrument matches multiple views, it can generate multiple metric streams.

Will the user reading this understand what "metric streams" means in this context? If not maybe link to where that is defined.

Reply:
better to stick with Metrics.